tests/test_backend.py

```diff
@@ -659,6 +659,43 @@ def test_get(self):
         self.assertEqual(self.cache.get("key", default="default"), "default")
 
 
+class DjangoOmitExceptionsPriority1Tests(unittest.TestCase):
+    def setUp(self):
+        self._orig_setting = django_redis.cache.DJANGO_REDIS_IGNORE_EXCEPTIONS
+        django_redis.cache.DJANGO_REDIS_IGNORE_EXCEPTIONS = False
```

Comment:
As `DJANGO_REDIS_IGNORE_EXCEPTIONS` is a setting, can this use the same `override_settings()` context manager instead of manipulating the module directly?

Reply:
I fix test's in this PR https://github.com/niwinz/django-redis/pull/335/files